lib/application.js

```diff
@@ -61,9 +63,14 @@ class Application extends events.EventEmitter {
     }
   }
 
-  runScript(methodName, session) {
+  runScript(iname, ver, methodName, session) {
     const { sandbox } = session || this;
-    const script = this.api.get(methodName);
+    const iface = this.api[iname];
+    if (!iface) return null;
```

Comment:
IMO It will be better to have this throw specific error codes to simplify debugging\logging of error cases.

Reply:
If `Application.prototype.createScript` returns null we generate `not found` in `client.js`. I don't think that we need different error types like: interface not found, method not found.